pkg/controller/plan/kubevirt.go

```diff
@@ -686,6 +688,22 @@ func (r *KubeVirt) createPodToBindPVCs(vm *plan.VMStatus, pvcNames []string) err
 	return nil
 }
 
+// Sets KVM requirement to the pod and container.
+func (r *KubeVirt) setKvmOnPodSpec(podSpec *core.PodSpec) {
+	if *r.Plan.Provider.Source.Spec.Type == v1beta1.VSphere && !Settings.VirtV2vDontRequestKVM {
+		if podSpec.NodeSelector == nil {
+			podSpec.NodeSelector = make(map[string]string)
+		}
+		podSpec.NodeSelector["kubevirt.io/schedulable"] = "true"
+		container := &podSpec.Containers[0]
+		if container.Resources.Limits == nil {
```

Comment:
while on it, shouldn't it rather be Requests here?

Reply:
changed to requests.